src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -19,13 +19,14 @@
 import time
 import zipfile
 
+from google.cloud.ndb import model
```

Comment:
Using `model._entity_{to,from}_protobuf()` everywhere requires a bunch of lint-ignore comments, and does not ensure the right entity type is read from a protobuf. How about introducing some common helpers?

```
def model_to_protobuf(model: ndb.Model) -> entity_pb2.Entity:
  return ndb._model_to_protobuf(model)  # pylint: disable=protected-access

# Can be typed like this, but it's overkill without type checking.
from typing import Type
from typing import TypeVar

T = TypeVar('T', bound=ndb.Model)

def model_from_protobuf(entity: entity_pb2.Entity, model_type: Type[T]) -> T:
  model = ndb._model_from_protobuf(testcase)   # pylint: disable=protected-access
  assert isinstance(model, model_type)
  return model
```

We could even introduce specific helpers for each model type, e.g. `testcase_to_protobuf()`, but this is likely enough already.